TombEngine/Sound/sound.cpp

```diff
@@ -628,6 +628,15 @@ int Sound_EffectIsPlaying(int effectID, Pose *position)
 	return -1;
 }
 
+bool Sound_EffectIsPlaying(int effectID)
```

Comment:
Can rename to `IsSoundEffectPlaying()` to comply with conventions. It doesn't need to be an overload of the legacy function since it does something different.

Reply:
done :)